src/file.ts

```diff
@@ -104,11 +104,27 @@ export function toFileWithPath(file: FileWithPath, path?: string): FileWithPath
         });
     }
 
+    // Always populate a relative path so that even electron apps have access to a relativePath value
+    Object.defineProperty(f, 'relativePath', {
+        value: typeof path === 'string'
+            ? path
+            // If <input webkitdirectory> is set,
+            // the File will have a {webkitRelativePath} property
+            // https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory
+            : typeof webkitRelativePath === 'string' && webkitRelativePath.length > 0
+                ? webkitRelativePath
+                : `/${file.name}`, // prepend forward slash (/) to ensure consistancy when path isn't supplied.
+        writable: false,
+        configurable: false,
+        enumerable: true
+    })
```

Comment:
This is not too different from setting the path. Let's make this block a function and reuse it for setting the `path` and `relativePath`.

Also, since the path is relative, `/${file.name}` should probably be `./${file.name}`.